### ISSUE TITLE
[Feature request] Create and snooze reminder

### ISSUE THREAD

**Issue by erikgeurts** [leggett/simplify#782]
I very much like the new Reminder feature in v3.1, and I've already started using it. Before I simply sent an email to myself, this is much more elegant.

One useful addition might be that while creating the reminder, you could snooze that same reminder right away for a future date. That's something I tend to do with the "note to self" emails. Send it and then snooze it until (for example) tomorrow at 9AM or Saturday at 9AM.

Being able to create a reminder and snooze it right away from the same dialog would be awesome, a nice "set it and forget it" tool to make those quickly and easily.

**Comment by leggett** [comment]
I tried to do this (as well as adding the option to star on send) but couldn't figure out a way. I could try to look for the email and snooze it as soon as I see it but that isn't going to work every time. 

So for now, this is on my "support this if I ever ask for full API access" list. :(

**Comment by erikgeurts** [comment]
One way to achieve this is to make use of the "send later" (Schedule Send) feature, but that takes a lot of clicks just to set one reminder:
- click the "More send options" icon
- click Schedule send (which is the only option, so why Gmail decided to dedicate an entire button just to open one more button is a mystery)
- click one of the presets or click "Select date and time", and in that case a lot of clicking to set the desired future time.

The disadvantage is that the reminder will be sitting in the Scheduled label/folder all the time it's waiting to be sent, mixed in with regular mails that are also scheduled for future sending. Another disadvantage is that a filter that would apply a label to the reminder will not be executed until the time the reminder is actually sent.

I understand the drawbacks of asking for API access, but at some point the benefits might outweigh them.

**Comment by leggett** [comment]
Yes, this is another path I considered and it may be worth revisiting until I can do something more robust with API access.
